src/Core/Components/Radio/FluentRadio.razor.cs

```diff
@@ -0,0 +1,75 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Extensions;
+//using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// A Fluent Radio button component.
+/// </summary>
+/// <typeparam name="TValue">The type for the value of the radio button</typeparam>
+public partial class FluentRadio<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)] TValue> : FluentInputBase<TValue>, IFluentField
```

Comment:
I don't think you need to implement `IFluentField`. Because the dev could use all properties like `LabelPosition`, `Message`, `MessageIcon` ... and without effect :-(